README.md

```diff
@@ -1,6 +1,53 @@
 # terraform-aws-vpc
 
-Terraform module that defines a VPC with public/private subnets across multiple AZs with Internet Gateways
+Terraform module that defines a VPC with Internet Gateway
+
+
+## Usage
+
+```terraform
+module "vpc" {
+  source    = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=master"
+  name      = "${var.name}"
+  namespace = "${var.namespace}"
+  stage     = "${var.stage}"
+}
```

Comment:
Let's add an example below this of adding subnets to the VPC with `terraform-aws-dynamic-subnets` just to be more friendly